lib/ex_ftx/http_client.ex

```diff
@@ -107,7 +107,11 @@ defmodule ExFtx.HTTPClient do
         request_body
       )
 
-    ["FTX-KEY": credentials.api_key, "FTX-SIGN": signature, "FTX-TS": timestamp]
+    if credentials.sub_account != nil do
```

Comment:
This works. But how do you feel about DRY'ing it up by extracting a function?

e.g.

```elixir
defp auth_headers(http_method, uri, request_body, credentials) do
   # ...
   ["FTX-KEY": credentials.api_key, "FTX-SIGN": signature, "FTX-TS": timestamp]
   |> put_sub_account_auth_header(credentials.sub_account)
end

defp put_sub_account_auth_header(headers, nil), do: headers
defp put_sub_account_auth_header(headers, sub_account), do: headers |> Keyword.put(:"FTX-SUBACCOUNT", sub_account)
```

Reply:
Yeah, this is way cleaner, updated